### PR TITLE
Activate AI Chapter

### LINKED ISSUE #3: Add diagram for Turing Test
Add a diagram for the Turing Test section in the AI chapter. Example can be found on [wikipedia.org](http://en.wikipedia.org/wiki/File:Turing_Test_version_3.png is an example).

### LINKED ISSUE #113: Fix formatting + check content for Artificial Intelligence

### PR DESCRIPTION
**Pull request by eAlasdair**
This PR activates the Artificial Intelligence chapter

Resolves #113 (from what I can tell)
Resolves #3 

## Checks

- [x] Full internationalization support

## Changes:

- [x] Content fixes
- [x] Replace screenshots of chatbots with more up to date versions
- [x] Add Turing test diagram image from [here](https://en.wikipedia.org/wiki/File:Turing_Test_version_3.png)

## Notes:

- The Alice chatbot (heavily referenced in the chapter) was missing, and has been replaced with another, seemingly equal Alice chatbot
- There are no interactives in the chapter
- Images are all in English as they are screenshots of the various chatbots and comic images from other sources
- Content talked about in #159 still needs to be implemented - I agree that it needs to be done, the chapter is sparse without it
- The AI sticks game from #574 still needs to be implemented

### CODE REVIEW

File: csfieldguide/chapters/content/en/artificial-intelligence/sections/chatterbots-and-the-turing-test.md [hunk 1]
```diff
@@ -11,28 +11,27 @@ You may come across chatterbots online for serious uses (such as giving help on
...
 {panel end}
 
-
 One such chatterbot is **Eliza**.
-Eliza attempts to sound like a Rogerian Psychotherapist, and sounds most intelligent in the kind of conversation that would be had in such a therapy session. ([Rogerian Psychotherapy](https://en.wikipedia.org/w/index.php?title=Rogerian_psychotherapy) is a technique that revolves around trying to understand the client by asking questions and reflecting their statements back to clarify that they've been understood).
+Eliza attempts to sound like a Rogerian Psychotherapist, and sounds most intelligent in the kind of conversation that would be had in a therapy session ([Rogerian Psychotherapy](https://en.wikipedia.org/w/index.php?title=Rogerian_psychotherapy) is a technique that revolves around trying to understand the client by asking questions and reflecting their statements back to clarify that they've been understood).
 
 {panel type="caution"}
 
 # Chatbots and personal information!"}
```
Comment (courtneycb) [inline csfieldguide/chapters/content/en/artificial-intelligence/sections/chatterbots-and-the-turing-test.md]: trailing `"}` needs to be removed

File: csfieldguide/chapters/content/en/artificial-intelligence/sections/chatterbots-and-the-turing-test.md [hunk 2]
```diff
@@ -103,7 +102,7 @@ If you want to know more, just ask Alice how she works, and she’ll tell you!
 
 Some possible rules that Alice might have are:
 
-- If "favourite colours" are mentioned,  say your favourite colour is "green".
+- If "favourite colours" are mentioned, say your favourite colour is "green".
 - If family members are mentioned (e.g. "mum", "dad", or "brother") say something that is about family.
```
Comment (courtneycb) [inline csfieldguide/chapters/content/en/artificial-intelligence/sections/chatterbots-and-the-turing-test.md]: Should we capitalise mum, dad, etc here?

File: csfieldguide/chapters/content/en/artificial-intelligence/sections/chatterbots-and-the-turing-test.md [hunk 3]
```diff
@@ -114,24 +113,29 @@ That’s why the replies sometimes use part of what you typed.
 
 The quality of these rules determines how intelligent the chatterbot will sound.  
 
-While Alice is undoubtedly more intelligent than Eliza, Alice still has her limitations. Sometimes she even admits to those limitations!
+While Alice is undoubtedly more intelligent sounding than Eliza, Alice still has her limitations.
+Sometimes she even admits to those limitations!
 
-**Judge**: What will happen in the year 2000  
+**Judge**: What will happen in the year 2000?
```
Comment (courtneycb) [inline csfieldguide/chapters/content/en/artificial-intelligence/sections/chatterbots-and-the-turing-test.md]: Need to add back in the new line
image

File: csfieldguide/chapters/content/en/artificial-intelligence/sections/chatterbots-and-the-turing-test.md (hunk 1 above)
Comment (eAlasdair) [reply csfieldguide/chapters/content/en/artificial-intelligence/sections/chatterbots-and-the-turing-test.md]: typo, thanks

File: csfieldguide/chapters/content/en/artificial-intelligence/sections/chatterbots-and-the-turing-test.md (hunk 2 above)
Comment (eAlasdair) [reply csfieldguide/chapters/content/en/artificial-intelligence/sections/chatterbots-and-the-turing-test.md]: They aren't used specifically as pronouns here so I don't think so

File: csfieldguide/chapters/content/en/artificial-intelligence/sections/chatterbots-and-the-turing-test.md (hunk 3 above)
Comment (eAlasdair) [reply csfieldguide/chapters/content/en/artificial-intelligence/sections/chatterbots-and-the-turing-test.md]: Not entirely sure what that character is, but I copy-pasted it back in

**Comment by eAlasdair** [comment]
> I reckon we could recreate the chatbot screenshots as uninteractives, what do you think?

The chatbots are actual websites, and uninteractives will undoubtedly look different, so there are potentially ethical and legal problems with doing so. Also the websites don't necessarily support the languages the uninteractives will be translated to

**Comment by courtneycb** [comment]
> The chatbots are actual websites, and uninteractives will undoubtedly look different

Yeah at first I thought we could just create them not based from any particular websites but the content references the sites heavily.. 

After checking out the sites I've had another thought. Can we update the images of the sites to more recent ones? 

E.g cleverbot old vs. new:
image

image

**Comment by JackMorganNZ** [comment]
> Also the websites don't necessarily support the languages the uninteractives will be translated to

This should probably be raised as an issue to check if the content is suitable for translation.

**Pull request merged**